config/jobs/kubernetes-sigs/kube-storage-version-migrator/kube-storage-version-migrator-presubmits.yaml

```diff
@@ -98,7 +98,7 @@ presubmits:
       - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230222-b5208facd4-master
         command:
         - runner.sh
-        - "./test/e2e/test-kinder.sh"
+        - "../kubeadm/kinder/ci/kinder-run.sh"
```

Comment:
This seems like the wrong path, given the path_aliases. Won't you need ../../k8s.io/kubeadm?

Reply:
I was following the  examples in other jobs, such as [this one](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-external-ca.yaml), which use the same `path_alias`.

Reply:
so the working directory is the first extra_refs for periodics, for presubmit and periodics it's the repo under test

in this case that means for this job the working dir will be the kube-storage-vesrsion-migrator repo clone which is at sigs.k8s.io/kube-storage-version-migrator

(both will be something like $GOPATH/src/$path_alias)

Reply:
i'm trying to find the relevant documentation but it's been moved in bulk recently and i've lost track of where this is covered if anywhere. it's a little confusing.

Reply:
Aha! https://docs.prow.k8s.io/docs/components/pod-utilities/#what-the-test-container-can-expect

Reply:
Updated. Thanks!